frontend/plugins/frontline_ui/src/modules/ticket/components/KanbanCard.tsx

```
@@ -0,0 +1,83 @@
+} from '@tabler/icons-react';
+import { format } from 'date-fns';
+import { Badge, Button, CalendarNew, Popover, Separator } from 'erxes-ui';
+import { useAtomValue } from 'jotai';
+import { currentUserState, MembersInline } from 'ui-modules';
+
+export const KanbanViewCard = ({ feature }: { feature: any }) => {
+  const currentUser = useAtomValue(currentUserState);
+  return (
+    <>
+      <div className="flex items-center justify-between h-9 px-1.5">
+        <KanbanDatePicker date={new Date()} onChange={() => {}} />
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)